src/platform/monitoring/external-services/reducer.js

```diff
@@ -13,9 +13,21 @@ export default function externalServiceStatuses(state = INITIAL_STATE, action) {
   switch (action.type) {
     case LOADING_BACKEND_STATUSES:
       return { ...state, loading: true };
-
+    // drive with external config with window
     case FETCH_BACKEND_STATUSES_FAILURE:
-      return { ...state, loading: false };
+      return {
+        ...state,
+        loading: false,
+        statuses: action.globalDowntimeActive
+          ? [
+              {
+                service: 'Global',
+                serviceId: 'global',
+                status: 'maintenance',
```

Comment:
if there is an active global downtime, then add the global status to backend statuses

Reply:
Should we be using the `global` constant here instead of the string value?